client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -37,16 +38,14 @@ export default React.createClass( {
 	render() {
 		let title;
 
-		if ( this.props.postViewsList.response.post && this.props.postViewsList.response.post.post_title ) {
-			title = this.translate( 'Stats for %(posttitle)s', {
-				comment: 'Title of the individual post stats page.',
-				args: {
-					posttitle: this.props.postViewsList.response.post.post_title
-				}
-			} );
-		}
+		const post = this.props.postViewsList.response.post;
+		const postOnRecord = post && post.post_title !== null;
 
-		if ( this.props.postViewsList.isError() ) {
+		if ( postOnRecord && ! this.props.postViewsList.isError() ) {
```

Comment:
> Could this if logic just check for the truthyness of `postOnRecord`

I checked out where it was added (1900-gh-calypso-pre-oss) for some context, but, it does look like we could just rely on `postOnRecord` here. I'll try taking the `isError` check out & make sure it still acts correctly.